go.mod

```diff
@@ -3,46 +3,113 @@ module github.com/ava-labs/teleporter
 go 1.18
 
 require (
-	github.com/ava-labs/avalanchego v1.10.9 // indirect
+	github.com/ava-labs/avalanchego v1.10.9
 	github.com/supranational/blst v0.3.11 // indirect
 )
 
 require (
+	github.com/ava-labs/avalanche-network-runner v1.7.2-0.20230825150237-723bc7b31724
+	github.com/ava-labs/awm-relayer v0.2.1-0.20230920145222-b8571020bdaa
```

Comment:
We can remove this dependency on `awm-relayer` in a future PR by migrating the used parts of `package teleporter` in `awm-relayer` to `teleporter`. We use the ABI packing/unpacking methods defined there, so it would make sense to have those live in the same repo as the `TeleporterMessenger` contract from which they are derived.